live-examples/js-examples/array/array-groupby.js

```diff
@@ -0,0 +1,13 @@
+const inventory = [
+  {name: 'apples', type: 'vegetables', quantity: 5},
```

Comment:
Why are apples vegetables? Maybe asparagus?

Reply:
Because I wasn't careful enough. Good suggestion - I like asparagus :-)
```suggestion
  {name: 'asparagus', type: 'vegetables', quantity: 5},
```